physics/lsm_ruc.meta

```diff
@@ -1527,6 +1621,7 @@
   type = real
   kind = kind_phys
   intent = inout
+  active = (control_for_land_surface_scheme == identifier_for_ruc_land_surface_scheme)
```

Comment:
The active attribute should only be added to host-side metadata.

Reply:
Removed.